iaso/diffing/exporter.py

```diff
@@ -233,6 +233,15 @@ def apply_comparison(self, payload, comparison):
         if comparison.field == "parent" and comparison.after:
             self.fill_parent_id(comparison, payload)
             return
+
+        if comparison.field == "opening_date":
+            payload["openingDate"] = comparison.after.strftime("%Y-%m-%dT%H:%M:%S")
+            return
+
+        if comparison.field == "closed_date":
+            payload["closedDate"] = comparison.after.strftime("%Y-%m-%dT%H:%M:%S")
```

Comment:
Is the format guaranteed here? Or should we catch a potential `ValueError`?

Reply:
I think there is no need to catch a potential ValueError as the date is from DHIS2 and the format it will stay the same